frog/imports/ui/ActivityEditor.jsx

```diff
@@ -126,7 +126,7 @@ class ActivityBody extends Component {
 
 export default createContainer(() => {
   return {
-    activities: Activities.find({}).fetch(),
-    operators: Operators.find({}).fetch(),
+    activities: Activities.find({status:'OUT'}).fetch(),
+    operators: Operators.find({status:'OUT'}).fetch(),
```

Comment:
Added to the todo list. I even want to get rid of it and make the difference on wether an activity has a field `graphId` or not